README.md

```diff
@@ -1,3 +1,5 @@
+![Frame 1Britany banner](https://user-images.githubusercontent.com/89431059/194469940-b753d9d3-7d63-4b22-b144-76afbf228a92.png)
```

Comment:
can we make the text something like `Hacktoberfest themed banner with repo name`